File: projects/order-bc-query/app/src/main/java/com/robinfood/app/mappers/orderstatushistory/OrderStatusMapper.java
package com.robinfood.app.mappers.orderstatushistory;

import com.robinfood.core.dtos.orderstatushistory.OrderStatusDTO;
import com.robinfood.core.entities.StatusEntity;

public class OrderStatusMapper {

    public static OrderStatusDTO orderStatusEntityToDTO(StatusEntity statusEntity) {

        return OrderStatusDTO.builder()
                .code(statusEntity.getCode())
                .id(statusEntity.getId())
                .build();
    }
}

File: projects/posv2/core/src/main/kotlin/com/robinfood/core/mappers/OrderStatusMapper.kt
package com.robinfood.core.mappers

import com.robinfood.core.dtos.OrderStatusDTO
import com.robinfood.core.entities.OrderStatusEntity

fun OrderStatusEntity.toOrderStatusDTO(): OrderStatusDTO {
    return OrderStatusDTO(
            id = id,
            name = name
    )
}